File: bpre/src/main/java/my/sample/bpre/api/Item.java
package my.sample.bpre.api;

import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlType;

/**
 * @author mohammad shamsi <m.h.shams@gmail.com>
 */
@XmlRootElement
@XmlType(propOrder = {"id", "name", "desc"})
public class Item {
    String id;
    String name;
    String desc;

    public Item() {
    }

    public Item(String id, String name, String desc) {
        this.id = id;
        this.name = name;
        this.desc = desc;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }
}

File: model/src/main/kotlin/my/sample/model/api/Item.kt
package my.sample.model.api

import javax.persistence.Entity
import javax.persistence.Id
import javax.xml.bind.annotation.XmlRootElement
import javax.xml.bind.annotation.XmlType

/**
 * @author mohammad shamsi <m.h.shams@gmail.com>
 */
@Entity
@XmlRootElement
@XmlType(propOrder = arrayOf("id", "name", "desc"))
open public class Item() {
    @Id
    public var id: String = ""
    public var name: String = ""
    public var desc: String = ""

    public constructor(id: String, name: String, desc: String) : this() {
        this.id = id
        this.name = name
        this.desc = desc
    }
}